.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: PyPI Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Setup dependencies
+        uses: ./.github/actions/setup-dependencies
+
+      - run: poetry publish --build
+        env:
+          POETRY_PYPI_TOKEN_PYPI: ${{secrets.POETRY_PYPI_TOKEN_PYPI}}
```

Comment:
This secret is not org level so we need to either set it at org level or put it into this repo. I'll keep you posted.

Reply:
@martonilles is on it. Please mark this thread as resolved when done :)

Reply:
Added POETRY_PYPI_TOKEN_PYPI secrets that can be used to publish to pypi